great_expectations/checkpoint/checkpoint.py

```diff
@@ -382,10 +378,7 @@ def _run_validation(
             )
             checkpoint_identifier = None
             if self.data_context.ge_cloud_mode:
-                checkpoint_identifier = GeCloudIdentifier(
-                    resource_type=GeCloudRESTResource.CHECKPOINT,
-                    ge_cloud_id=str(self.ge_cloud_id),
-                )
+                checkpoint_identifier = self.ge_cloud_id
```

Comment:
nit: we probably want to rename this to `checkpoint_id` here and where it's passed as an argument to avoid confusion with `GeCloudIdentifier` but I don't know that this is pressing